spec/models/fact_spec.rb

```diff
@@ -82,6 +86,49 @@
       let (:stats) { {"mean" => 0.5, "length" => 3, "stdev" => 1} }
       it { is_expected.to_not be_valid }
     end
+
+    context 'with exported space' do
+      let (:exporting_space) { FactoryGirl.create(:space) }
+      let (:metric_id) { nil }
+
+      it { is_expected.to_not be_valid }
+
+      context 'with a metric id' do
+        let (:metric_id) { '3' }
+
+        before do
+          exporting_space
+          expect(exporting_space.exported_facts_count).to be 0
+        end
+
+        it { is_expected.to be_valid }
+
+        it 'should increment the exporting_space exported_facts_count upon creation' do
+          fact.save
+          exporting_space
```

Comment:
I believe the ``exporting_space`` line in the ``before do`` would happen here.  It seems strange if it needs to happen twice.